farmhelper-mod/src/main/java/com/jelly/farmhelper/network/proxy/ProxyManager.java

```diff
@@ -19,25 +19,30 @@
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 
+
+
 public class ProxyManager {
     public static boolean isTesting = false;
 
+    public enum ProxyType {
```

Comment:
You've added HTTP and HTTPS to the `ProxyType.java` but here you didn't add it for some reason? 🤔

Reply:
I plan to do it, is it okay if I leave it?

Reply:
As long as it's not visible to user through GUI it's okey, I thought you forgot about it

Reply:
kay :D